### PR TITLE
Hugetlb enable

### PR DESCRIPTION
**Pull request by JasonChenCJ**

### CODE REVIEW

File: hypervisor/Documentation/ACRN_UEFI.txt [hunk 1]
```diff
@@ -11,3 +11,10 @@ d. Update the bsp/uefi/clearlinux/acrn.conf file by filling the field <UUID of r
 e. Copy bsp/uefi/clearlinux/acrn.conf to loader\\entries\\ directory.
 f. Reboot.
 g. If you want to recover to original os loader, just replace the bootloaderx64.efi with bootloaderx64_origin.efi.
+
+Note:
```
Comment (gvancuts) [inline hypervisor/Documentation/ACRN_UEFI.txt]: Thanks for taking the time to write these notes @JasonChenCJ 

This file will very shortly be deleted however as *all* documentation should be put under `doc/`. You could enhance [this section](https://github.com/projectacrn/acrn-hypervisor/blob/master/doc/primer/index.rst#memory-virtualization) in the Developer Primer or create a new tech note under the "How-To" section.

File: hypervisor/Documentation/ACRN_UEFI.txt [hunk 2]
```diff
@@ -11,3 +11,10 @@ d. Update the bsp/uefi/clearlinux/acrn.conf file by filling the field <UUID of r
...
+There are two kind of DM memory allocation mechanism - CMA & Hugetlb, they can be choose but hugetlb will become default one.
+- To support CMA, please make sure:
+  a. add "cma=reserved_mem_size@recommend_memory_offset-0" like "cma=2560M@0x100000000-0" to the SOS cmdline in acrn.conf
+  b. start acrn-dm without "-T" option
+- To support Hugetlb, please refer to Documentation/ACRN_HUGETLB.txt
```
Comment (gvancuts) [inline hypervisor/Documentation/ACRN_UEFI.txt]: Did you forget to add this ACRN_HUGETLB.txt file to the commit? (make sure to put it under `doc/`)

**Comment by JasonChenCJ** [comment]
@gvancuts , I updated the second commit, please help review :)

**Comment by gvancuts** [comment]
Thanks @JasonChenCJ ! I think we only really need to keep the section called "DM memory allocation mechanism" (the one before is a repeat from the Getting Started Guide so no need to duplicate). I'm thinking that @dbkinder will most likely do a final pass on this and I would suggest that we merge this and do the final pass later on. Does that work for you @dbkinder ?

**Comment by lijinxia** [comment]
verified +1

**Pull request merged**